data/items.js

```diff
@@ -4035,6 +4035,66 @@ exports.BattleItems = {
 		gen: 3,
 		desc: "Cannot be eaten by the holder. No effect when eaten with Bug Bite or Pluck.",
 	},
+	"poweranklet": {
+		id: "poweranklet",
+		name: "Power Anklet",
+		isUnreleased: false,
```

Comment:
Ah, I see. I didn't know what Kris was referring to and I'm new to Github so I wasn't sure if there was something I was missing. In retrospect, that should have been obvious. Sorry! Updating pull request now.